css/project-list-pages.styl

```diff
@@ -43,6 +43,10 @@
     .pill-button
       margin-right: 10px
       padding: 1em
+      color: rgb(64,64,64)
+      font-weight: normal
+      border-color: rgb(64,64,64) !important
```

Comment:
I had a quick look and I've found the problem. There's an error in the markup that I submitted as part of this commit 2786572acef9bccba20e45f074b7700d115c7878

I'm happy to fix the markup (the pagination stuff needs moving out of the `resource-results-counter` div) but you  don't need to submit any CSS changes in this PR.